Datto-RMM/scripts/InstallHuntress.dattormm.comstore.ps1

```diff
@@ -873,6 +899,19 @@ function copyLogAndExit {
 #                                  begin main function                                  #
 #########################################################################################
 function main () {
+    if ($env:repairAgent -eq $true) {
+        $repair = $true
+    }
+    if ($env:reregisterAgent -eq $true) {
+        $reregister = $true
+    }
+    if ($env:uninstallAgent -eq $true) {
```

Comment:
This looks like the only variable being used from here... are repair, reregister, and reinstall needed?

Reply:
`repairAgent`, `reregisterAgent`, `uninstallAgent`, and `reinstallAgent` are environment variables set by Datto before calling the PowerShell script, these environment variables toggle the corresponding `repair`, `reregister`, `uninstall`, and `reinstall` variables that would typically be passed to the script as command-line arguments and used further down in the `main` function.

Reply:
This was unintentionally removed for `reinstallAgent` and `reregisterAgent` in a previous PR: https://github.com/huntresslabs/deployment-scripts/blob/eaa66a00fe1cc7bd94706e75ec6f93bc0e30bf14/Datto-RMM/scripts/InstallHuntress.dattormm.comstore.ps1#L419